fs/layer/layer.go

```diff
@@ -103,6 +105,9 @@ type Layer interface {
 	// ReadAt reads this layer.
 	ReadAt([]byte, int64, ...remote.Option) (int, error)
 
+        //True if none of the files in the layer has an xattr
+        NoXAttr() bool
```

Comment:
nit: I think the positive version is easier to read. I'm ok with it as is if there's pushback on that.
```suggestion
        HasXattrs() bool
```

Reply:
I agree that it's easier to read, but I think there is maintenance and bug prevention value in arranging it for the default to be False so that things Just Work when we default-construct something containing this bool.